README.md

```diff
@@ -157,6 +170,16 @@ Ecto.Multi.new()
 |> MyApp.Repo.transaction()
 ```
 
+Using the `enqueue/3` function the `:idempotency_key` works as expected:
```

Comment:
Perhaps "the `:idempotency_key` works as expected:" should say "the `:idempotency_key` is used to deduplicate jobs"